openpower/configs/linux/skiroot_defconfig

```diff
@@ -1,5 +1,7 @@
 CONFIG_PPC64=y
 CONFIG_ALTIVEC=y
+CONFIG_PPC_POWERNV_RTAS=n
+CONFIG_PPC_PSERIES=n
```

Comment:
Only if you don't use @ozbenh powernv qemu. Which we should probably move to. Only used in petitboot-tests atm.